zomes/hc_zome_trust_atom/src/test_helpers.rs

```diff
@@ -35,7 +47,36 @@ pub fn create_string_target(input: String) -> ExternResult<EntryHash> {
   Ok(target_entry_hash)
 }
 
-fn link_tag(tag: String) -> ExternResult<LinkTag> {
-  let serialized_bytes: SerializedBytes = StringLinkTag(tag).try_into()?;
-  Ok(LinkTag(serialized_bytes.bytes().clone()))
+pub fn create_test_entry(input: Test) -> ExternResult<HeaderHash> {
+  create_entry(input)
+}
+
+pub fn get_entry_by_header(header_hash: HeaderHash) -> ExternResult<Test> {
+  let element = get_element_by_header(header_hash, GetOptions::default())?;
+  match element.entry() {
+    element::ElementEntry::Present(entry) => {
+      Test::try_from(entry.clone()).or(Err(WasmError::Guest(format!(
+        "Couldn't convert Element entry {:?} into data type {}",
+        entry,
+        std::any::type_name::<Test>()
+      ))))
+    }
+    _ => Err(WasmError::Guest(format!(
+      "Element {:?} does not have an entry",
+      element
+    ))),
+  }
+}
+#[warn(clippy::pedantic)]
```

Comment:
Hmm could you allow a more specific linter rather than a class of linters?  ie pedantic is a class.